packages/webapp/pages/onboarding.tsx

```diff
@@ -101,9 +110,21 @@ export function OnboardPage(): ReactElement {
   const onSuccessfulTransaction = () => {
     onShouldUpdateFilters(true);
     setFinishedOnboarding(true);
+    if (hasCategories) {
+      return;
+    }
```

Comment:
When does this happen? Why are we stopping the redirection from here?

Reply:
We are doing this to ensure if the user has set categories we wait for their feed to be created before redirecting them so the page doesn't just going to popular then my-feed

Reply:
Right. Good catch. Thanks for the clarification.